context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -209,18 +201,20 @@ protected final void startWatcher() {
 
             @Override
             public void run() {
-                this.watchService = FileSystems.getDefault().newWatchService();
+                try {
+                    this.watchService = FileSystems.getDefault().newWatchService();
+                } catch (IOException e) {
```

Comment:
`newWatchService` may also throw `UnsupportedOperationException` - shouldn't this be handled as well?

Comment also applies to the other two file monitor classes.